developer-docs-site/docs/concepts/coin-and-token/mint-minting-tool.md

```diff
@@ -0,0 +1,91 @@
+# NFT Minting Tool CLI
+
+> The tool is tested on Mac. It should work on Linux and WSL too.
+
+# Setup
+
+## Node and NPM
+
+Node is required to run the minting CLI. If nodejs is not available on your OS, make sure to install that first. You can download nodejs from here [https://nodejs.org/en/download/](https://nodejs.org/en/download/).
+
+## Get the Aptos CLI
+
+Install the CLI following this guide [https://aptos.dev/cli-tools/aptos-cli-tool/install-aptos-cli](https://aptos.dev/cli-tools/aptos-cli-tool/install-aptos-cli). Aptos CLI is needed for compiling and deploying the minting contract.
+
+Note: on Mac, Aptos CLI can be installed with `brew install aptos`
```

Comment:
Let's just refer to the install-aptos-cli, so we don't have repeated information about how to install.